QUICKSTART.md

```diff
@@ -25,7 +25,7 @@ Before running, make sure you have installed and appropriately-configured the fo
 * [Helm 3](https://helm.sh/) for deploying to Kubernetes and managing deployments
 
 ## Running the Quickstart Script
-Run the cross-platform [quickstart.cmd script](quickstart.cmd):
+Run the cross-platform [quickstart.cmd script](/quickstart.cmd):
```

Comment:
These need to be the full web address to the scripts since the gitpages are served from the /docs folder and the scripts are outside that folder. So it will need to be https://github.com/gchq/Palisade/blob/main/quickstart.cmd

Reply:
done

Reply:
fixed